feature/experimental/policy/policy_vrf_selection/ate_tests/base_vrf_selection/base_vrf_selection_test.go

```diff
@@ -155,100 +167,103 @@ func configInterfaceDUT(i *oc.Interface, me, peer *attrs.Attributes, subintfinde
 }
 
 // Configure Network instance on the DUT
-func configNetworkInstance(name string, peer *attrs.Attributes) *oc.NetworkInstance {
+func configNetworkInstance(name string, intf *oc.Interface, id uint32) *oc.NetworkInstance {
 	d := &oc.Root{}
 	ni := d.GetOrCreateNetworkInstance(name)
 
 	ni.Type = oc.NetworkInstanceTypes_NETWORK_INSTANCE_TYPE_L3VRF
-	static := ni.GetOrCreateProtocol(oc.PolicyTypes_INSTALL_PROTOCOL_TYPE_STATIC, *deviations.StaticProtocolName)
-	ipv4Nh := static.GetOrCreateStatic("0.0.0.0/0").GetOrCreateNextHop("0")
-	ipv4Nh.NextHop, _ = ipv4Nh.To_NetworkInstance_Protocol_Static_NextHop_NextHop_Union(peer.IPv4)
-	ipv6Nh := static.GetOrCreateStatic("::/0").GetOrCreateNextHop("0")
-	ipv6Nh.NextHop, _ = ipv6Nh.To_NetworkInstance_Protocol_Static_NextHop_NextHop_Union(peer.IPv6)
-	ipv4Nh.Recurse = ygot.Bool(true)
-	ipv6Nh.Recurse = ygot.Bool(true)
+	niIntf := ni.GetOrCreateInterface(*intf.Name)
+	niIntf.Subinterface = ygot.Uint32(id)
+	niIntf.Interface = ygot.String(*intf.Name)
 
 	return ni
 }
 
+// configDefaultRoute configures a static route in DEFAULT network-instance.
+func configDefaultRoute(t *testing.T, dut *ondatra.DUTDevice, v4Prefix, v4NextHop, v6Prefix, v6NextHop string) {
+	t.Logf("*** Configuring static route in DEFAULT network-instance ...")
+	ni := oc.NetworkInstance{Name: deviations.DefaultNetworkInstance}
+	static := ni.GetOrCreateProtocol(oc.PolicyTypes_INSTALL_PROTOCOL_TYPE_STATIC, *deviations.StaticProtocolName)
+	sr := static.GetOrCreateStatic(v4Prefix)
+	nh := sr.GetOrCreateNextHop("0")
+	nh.NextHop = oc.UnionString(v4NextHop)
+	gnmi.Update(t, dut, gnmi.OC().NetworkInstance(*deviations.DefaultNetworkInstance).Protocol(oc.PolicyTypes_INSTALL_PROTOCOL_TYPE_STATIC, *deviations.StaticProtocolName).Config(), static)
+	sr = static.GetOrCreateStatic(v6Prefix)
+	nh = sr.GetOrCreateNextHop("0")
+	nh.NextHop = oc.UnionString(v6NextHop)
+	gnmi.Update(t, dut, gnmi.OC().NetworkInstance(*deviations.DefaultNetworkInstance).Protocol(oc.PolicyTypes_INSTALL_PROTOCOL_TYPE_STATIC, *deviations.StaticProtocolName).Config(), static)
+}
+
+// configvrfRoute configures a static route.
```

Comment:
You don't need comments on unexported functions, but if you create them, recommended to start with the exact function name. So either drop this comment or start with exactly "configVRFRoute"

Reply:
done. in new commit.,